src/tracks/playabletrack/wavetrack/ui/WaveClipAdjustBorderHandle.cpp

```diff
@@ -376,34 +397,63 @@ class AdjustBetweenBorders final : public WaveClipAdjustBorderHandle::AdjustPoli
 };
 
 
-HitTestPreview WaveClipAdjustBorderHandle::HitPreview(const AudacityProject*, bool unsafe)
+HitTestPreview WaveClipAdjustBorderHandle::HitPreviewTrim(const AudacityProject*, bool unsafe, bool isLeftBorder)
 {
     static auto disabledCursor =
```

Comment:
All these declarations could also be `const` it would seem.

Reply:
This duplicates similar functions among other handles, in cases where I write something by-analogue I usually prefer to "copy" code-style as well. But this note is more about constness of copied-by-value argument, right? From what I see it's not a common practice in Audacity's code